src/rgw/rgw_log.cc

```diff
@@ -128,14 +128,13 @@ class UsageLogger {
     round_timestamp = ts.round_to_hour();
   }
 
-  void insert_user(utime_t& timestamp, const rgw_user& user, rgw_usage_log_entry& entry) {
+  void insert_user(utime_t& timestamp, const string& user, rgw_usage_log_entry& entry) {
```

Comment:
The convection of `<tenant>:<user>` has been added intensionally and it's extensively used across the code. I would prefer to not change that.